src/providers/ldap/sdap_async.c

```diff
@@ -2414,10 +2421,18 @@ static void sdap_sd_search_done(struct tevent_req *subreq)
                                     &state->ref_count,
                                     &state->refs);
     talloc_zfree(subreq);
+
     if (ret != EOK) {
-        DEBUG(SSSDBG_OP_FAILURE,
-              "sdap_get_generic_ext_recv failed [%d]: %s\n",
-              ret, sss_strerror(ret));
+        if (ret == ETIMEDOUT) {
+            DEBUG(SSSDBG_CRIT_FAILURE,
+                  "sdap_async_sys_connect request failed: [%d]: %s "
+                  "[ldap_network_timeout]\n",
+                  ret, sss_strerror(ret));
+        } else {
+            DEBUG(SSSDBG_CRIT_FAILURE,
+                  "sdap_async_sys_connect request failed: [%d]: %s\n",
```

Comment:
The function that is failing is called sdap_get_generic_ext_recv, not sdap_async_sys_connect